### PR TITLE
Document 'firetv.adb_command' service

### PR DESCRIPTION
**Pull request by JeffLIrion**
**Description:**

Documentation for the `media_player.firetv_adb_cmd` service.  

Generated documentation: https://deploy-preview-8722--home-assistant-docs.netlify.com/components/media_player.firetv/

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** https://github.com/home-assistant/home-assistant/pull/21419

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by JeffLIrion** [comment]
Does `media_player.firetv.markdown` need to be renamed, maybe to `firetv.markdown` or `firetv.media_player.markdown`?

**Comment by MartinHjelmare** [comment]
@fabaff should we move the page, since the platform has moved to embedded under the firetv component package? The configuration hasn't changed.

**Comment by JeffLIrion** [comment]
Any consensus on whether this file should be renamed? The documentation website builds just fine, so renaming this file could always be done later.

File: source/_components/media_player.firetv.markdown [hunk 1]
```diff
@@ -117,3 +117,21 @@ If you receive the error message `Issue: Error while setting up platform firetv`
 3. Home Assistant does not have the appropriate permissions for the `adbkey` file and so it is not able to use it.  Once you fix the permissions, the component should work.
 
 4. You are already connected to the Fire TV via ADB from another device.  Only one device can be connected, so disconnect the other device, restart the Fire TV (for good measure), and then restart Home Assistant.  
+
+
+## {% linkable_title Service %}
```
Comment (rohankapoorcom) [inline source/_components/media_player.firetv.markdown:122]: This is the old style of documenting services. Please use the new style. Look at [persistent_notification](https://raw.githubusercontent.com/home-assistant/home-assistant.io/next/source/_components/persistent_notification.markdown) for an example

File: source/_components/media_player.firetv.markdown [hunk 2]
```diff
@@ -117,3 +117,21 @@ If you receive the error message `Issue: Error while setting up platform firetv`
...
+
+A service `firetv.adb_command` is provided allowing you to send either keys or ADB shell commands to the Fire TV.
+
+```json
```
Comment (rohankapoorcom) [inline source/_components/media_player.firetv.markdown]: We normally write this in yaml.

**Review by rohankapoorcom — CHANGES_REQUESTED** [review]
A few changes requested to match style guidelines.

File: source/_components/media_player.firetv.markdown (hunk 1 above)
Comment (JeffLIrion) [reply source/_components/media_player.firetv.markdown:122]: Fixed

File: source/_components/media_player.firetv.markdown (hunk 2 above)
Comment (JeffLIrion) [reply source/_components/media_player.firetv.markdown]: Fixed

**Review by rohankapoorcom — APPROVED** [review]

**Comment by rohankapoorcom** [comment]
Thanks! I'm going to go ahead and merge this in since there's more changes coming to this file (#8806). If it needs to be renamed, we can handle it elsewhere.

**Pull request merged**